copter/source/docs/weathervaning.rst

```diff
@@ -0,0 +1,73 @@
+.. _weathervaning:
+
+Weathervaning and Wind Hold
+===========================
+
+Flying a Traditional Helicopter, or some Multicopters, in significant levels of wind can present a
+challenge. The issue is that any flat plate surface, such as a Traditional Helicopter's tail fin and boom, offers a lot of surface area for the wind to interact with. That can lead to a
+reduction in attitude and position control and high motor and ESC
+load.
+
+To reduce the impact of wind when flying in VTOL modes the ArduPilot code supports active yaw axis weathervaning which can greatly reduce the impact of wind on
+VTOL flight by keeping the vehicle pointed into the wind and reducing
```

Comment:
```suggestion
To reduce the impact of wind when flying in VTOL modes the ArduPilot code supports active yaw axis weathervaning which can greatly reduce the impact of wind by keeping the vehicle pointed into the wind and reducing
```

Reply:
should probably drop the VTOL bit here too since Copter and Heli ONLY can fly in VTOL modes